### PR TITLE
Fix Grammar & Formatting Issues

### PR DESCRIPTION
**Pull request by adamaveray**
Fixes a handful of grammar issues and reformats much of the content as proper Markdown (e.g. tables and lists).

### CODE REVIEW

File: tab_technical.md [hunk 1]
```diff
@@ -9,233 +9,170 @@ tags: headers
...
+### headers
 
 Python script to get some response headers from Alexa top sites file and store in a MySQL database.
 
+**Github:** https://github.com/oshp/headers/
 
-Github: https://github.com/oshp/headers/
-
-securityheaders.io
+### securityheaders.io
```
Comment (Malvoz) [inline tab_technical.md:28]: This site is now https://securityheaders.com/

**Comment by riramar** [comment]
WOW! Thanks so much @adamaveray ! I own you a beer :)

**Comment by riramar** [comment]
hey @adamaveray I hope you don't mind that I created the new section Contributors in the main page and put your name.
https://owasp.org/www-project-secure-headers/

**Comment by adamaveray** [comment]
Happy to help! 🙌

**Pull request merged**